KSystemInformer/cid_tracking.c

```diff
@@ -332,6 +334,15 @@ NTSTATUS KSIAPI KphpInitializeProcessContext(
         process->ImageFileName = NULL;
     }
 
+    if (KphUSrchr(process->ImageFileName, L'\\', &process->ImageName)) 
+    {   // skip L'\\'
+        process->ImageName.Buffer ++;
+        process->ImageName.Length -= sizeof(WCHAR);
+        process->ImageName.MaximumLength -= sizeof(WCHAR);
+    }
+    else
+        process->ImageName = KphpUnknownImageName;
```

Comment:
> CI fixed, style issue fixed,
about showing "UNKNOWN" for some minimal processes, what other string would you suggest, "UNNAMED"?

I think it might be most appropriate to just zero Unicode string rather than use "UNNAMED"/"UNKNOWN". I recognize this is just for logging at this time and showing an empty string might not look great in the log (the absence of a string is arguably equal to "UNKNOWN" anyway). I might have use for `ImageName` elsewhere, and having it reflect "has a value or is an empty string" is ideal I think. I guess what you have is fine for now. I'll probably just go in and adjust it later.